### PR TITLE
Assign mongodb group ownership to /var/lib/mongodb folder

### PR DESCRIPTION
**Pull request by narengan**
The files within this folder were assigned nogroup.

### CODE REVIEW

**Comment by lihkin213** [comment]
Can we also test this on RHOSP

**Comment by narengan** [comment]
retest

File: roles/mongodb-common/tasks/main.yml [hunk 1]
```diff
@@ -56,6 +56,18 @@
...
+    path: "/var/lib/mongodb"
+    group: mongodb
+    recurse: yes
+
+- name: Change permissions of /var/lib/mongodb folder
+  file:
+    path: "/var/lib/mongodb"
+    state: directory
+    group: root
+    mode: 0750
```
Comment (nirajdp76) [inline roles/mongodb-common/tasks/main.yml]: need to add new line

**Comment by nirajdp76** [comment]
ok to test

**Comment by lihkin213** [comment]
LGTM

File: roles/mongodb-common/tasks/main.yml (hunk 1 above)
Comment (narengan) [inline roles/mongodb-common/tasks/main.yml]: Added new line

**Pull request merged**